Fix for Issue #322 Cannot open links in boosted toots

danschwarz opened the pull request:
Issue #322 fixed with this PR

ihabunek commented:
Please reword this commit message in accordance with the [guidelines](https://toot.bezdomni.net/contributing.html#crafting-good-commits). You can add `fixes #322` in the commit message, this will make github close the issue upon merge.

danschwarz commented:
OK, fixed up the commit message. Also added a little extra in subsequent commit - deduplication and sorting of link URLs.  Feel free to keep or omit that.

ihabunek commented:
Merged manually since you left an empty commit I wanted to get rid of. :) Thanks!
